site/_posts/2017-10-30-crossword-lwimiq.html.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Crossword LWiMIQ"
+author: d-m-u
+date: 2017-10-30
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+## A Puzzle for your Perusal
+
+Any puzzles, correctly completed and returned to me before next week's blog, will enter you into a drawing for a prize!
+
+![crossword](/site/assets/images/blog/crossword.jpg)
+
+
+ACROSS  
+4	type of attributes returned only if specified in api  
+7	rubocop punctuation warnings removed for hacktoberfest  
+8	new name of ShowbackDataRollup in manageiq-consumption  
+15	thing being removed in favor of purge_in_batches (3 wds.)  
+19	what we're supporting for windows templates in main #16263  
+20	method that instance_profile creation in amazon prov includes  
+21	type that value passed into sleep method in azure armrest must be  
+23	name of branch on which unprivileged docker user login got fixed (5 wds.)  
+24	what happened to instructions on accessing import image screen for RHV  
+27	type of broken tests fixed in classic-ui very recently (2 wds.)  
+28	auth type that now includes documentation for ldap configmap generator  
+30	instance resize operation location in main #16044 (2 wds.)  
+31	series of characters that represent space removed for hacktoberfest  
+
+DOWN  
+1	what replaced dialog logic  
+2	userid purview, fixed this week in main #16323  
+3	what vsd_client in nuage now includes  
+5	type of driver enhancements added recently to middleware  
+6	div that got moved and deduplicated on settings region screen  
+9	location of event initiator  
+10	recently fixed elements on set ownership screen  
+11	type of size added to Switch model  
+12	name of method that got fixed in StorageAccountService (3 wds.)  
+13	kind of controller types now being listed in vmware prov repo  
+14	operation approval workflow now being used for (2 wds.)  
+16	api response immensity  
+17	new table in schema added at end of last week (3 wds.)  
+18	things now connected to targeted refresh in nuage providers  
+22	addition to mac guides involving the version of this  
+25	kind of event triggers added for Hawkular  
+26	what got updated for travis in the appliance console repo (2 wds.)  
+29	layout type in which MD and SM viewport sizes differ  
+
+
+
+ [//]: # (Credit for the puzzle idea goes to GM, credit for the contest idea goes to ABW.)
+ [//]: # (Yes, I'm a shameless stealer of ideas.)
```

Comment:
These two lines won't show up in the published post I think, is that your intention?

Anyway, brilliant idea for an LWIMIQ post, borrowed or otherwise! ;)

Reply:
Yup, that's what I wanted.